packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1728,6 +1742,15 @@ export default class NewVm extends BaseComponent {
                 value={cpuCap}
               />
             </Item>
+            <Item label={_('cpusMax')}>
+              <DebounceInput
+                className='form-control'
+                min={0}
+                onChange={this._linkState('cpusMax')}
+                type='number'
+                value={cpusMax}
+              />
+            </Item>
```

Comment:
Should we move the topology selector here, then?

Reply:
The cores per socket impacts the VM performance. So, IMHO, it's in the right place. Why we should move it here?